integrated_channels/canvas/client.py

```diff
@@ -44,7 +60,77 @@ def update_content_metadata(self, serialized_data):
     def delete_content_metadata(self, serialized_data):
         pass
 
-    def _get_oauth_access_token(self, client_id, client_secret):
+    def _post(self, url, data):
```

Comment:
why are these _post, _delete, etc. showing as new methods? Didn't you already create these in the client pr?

Reply:
I think this may be a rebase issue because I had branched this off of my last client PR and when that was merged github automatically switched target branches. I'll do a local rebase and I assume it will clear it up.